utils/api.js

```diff
@@ -97,6 +91,12 @@ const api = {
       }
     }
 
+    if (response.statusCode >= 400) {
+      throw new Error(
+        `${response.statusCode} ${response.body.substr(0, 100)}...`
```

Comment:
100 characters are enough to see the validation errors?

Reply:
LMAO I didn't see that :laughing: